react/features/follow-me/middleware.ts

```diff
@@ -124,7 +119,7 @@ function _onFollowMeCommand(attributes = {}, id, store) {
         const { conference } = state['features/base/conference'];
 
         // As this participant is not stored in redux store we do the checks on the JitsiParticipant from lib-jitsi-meet
-        const participant = conference.getParticipantById(id);
+        const participant = conference?.getParticipantById(id);
```

Comment:
we don't have a type `Participant`? Would it be complicated to create it?

Reply:
We do, but this isn't `Participant` it's `JitsiParticipant`. And `JitsiParticipant` should be a lib-jitsi-meet interface